k2/csrc/log.cuh

```diff
@@ -123,13 +127,32 @@ class Logger {
 
 class Voidifier {
  public:
-  __host__ __device__ void operator&(const Logger &)const {}
+  K2_CUDA_HOSTDEV void operator&(const Logger &) const {}
 };
 
 }  // namespace internal
 
 }  // namespace k2
 
+/**
+ * @brief A static assertion
+ *
+ * @param[in] COND the compile-time boolean expression that must be true
+ * @param[in] MSG  an error message if exp is false
+ *
+ * @note `static_assert` is supported by both of host and device.
```

Comment:
I think `static_assert` is never used in this file.

Reply:
yes, corrected